README.md

```diff
@@ -476,37 +608,26 @@ candidates_pd = \
                      number_test_rows=20)
 ```
 
-Note that the only difference compared to the previous command is `ranking_metric="conf"`.
 
-Similarly via command line:
+Note that the only difference compared to the previous command is `ranking_metric="cosine"`, or via command line:
 
 ```bash
-DeezyMatch --deezy_mode candidate_ranker -comb ./combined/test -rm conf -t 0.51 -n 1 -sz 4 -o test_candidates_deezymatch -mp ./models/finetuned_test001/finetuned_test001.model -v ./models/finetuned_test001/finetuned_test001.vocab -tn 20
+DeezyMatch --deezy_mode candidate_ranker -comb ./combined/test -rm cosine -t 0.51 -n 1 -sz 4 -o test_candidates_deezymatch -mp ./models/finetuned_test001/finetuned_test001.model -v ./models/finetuned_test001/finetuned_test001.vocab -tn 20
 ```
 
-* Select candidates based on cosine similarity:
 
-```python
-from DeezyMatch import candidate_ranker
+## Tips / Suggestions on DeezyMatch functionalities
 
-# Find candidates
-candidates_pd = \
-    candidate_ranker(scenario="./combined/test/", 
-                     ranking_metric="cosine", 
-                     selection_threshold=0.51, 
-                     num_candidates=1, 
-                     search_size=4, 
-                     output_filename="test_candidates_deezymatch", 
-                     pretrained_model_path="./models/finetuned_test001/finetuned_test001.model", 
-                     pretrained_vocab_path="./models/finetuned_test001/finetuned_test001.vocab", 
-                     number_test_rows=20)
-```
+### Candidate ranker
 
-Or via command line:
+* As already mentioned, based on our experiments, `conf` is not a good metric for ranking candidates. Consider using `faiss` or `cosine`.
+
+* Adding prefix/suffix to input strings (see `prefix_suffix` option in the input file) can greatly enhance the ranking results.
```

Comment:
I changed the line to:

```
Adding prefix/suffix to input strings (see prefix_suffix option in the input file) can greatly enhance the ranking results. However, we recommend one-character-long prefix/suffix; otherwise, this may affect the computation time.
```

Is that OK?